tests/jobs/npv/test_npv_cli.py

```diff
@@ -157,7 +157,7 @@ def test_npv_main_entry_point_warning_for_default_output(
     monkeypatch.setattr(
         parser,
         "add_summary_argument",
-        lambda p: p.add_argument("--summary", default=ecl_summary_npv()),
+        lambda p, **_: p.add_argument("--summary", default=ecl_summary_npv()),
```

Comment:
We need this because we now provide "**kwargs" for this method and it's only used in the test_npv_cli I guess?

Reply:
Yes that is the reason this is showing up here